web-admin/src/routes/+page.svelte

```diff
@@ -27,29 +27,31 @@
   <title>Home - Rill</title>
 </svelte:head>
 
-<AuthRedirect>
-  <section
-    class="flex flex-col mx-8 my-8 sm:my-16 sm:mx-16 lg:mx-32 lg:my-24 2xl:mx-64 mx-auto"
-  >
-    <h1 class="text-4xl leading-10 font-light mb-2">
-      Hi {getFirstNameFromDisplayName($user.data.user.displayName)}!
-    </h1>
-    <div class="flex flex-row gap-x-7 flex-wrap">
-      <div class="md:w-1/2">
-        {#if $orgs.isSuccess}
-          {#if !hasAnOrganization}
-            <WelcomeMessage />
-          {:else}
-            <h3 class="text-base leading-6 font-normal text-gray-500 mb-3">
-              Check out your dashboards below.
-            </h3>
-            <OrganizationList />
+<div class="overflow-auto h-full">
```

Comment:
Given you're using this in multiple places, what about making this a component `VerticalScrollContainer` (or something similarly named)?

```svelte
<div class="overflow-auto h-full">
    <slot />
</div>
```

Then, in this case, it'd be clear that the `<AuthRedirect>` should be a parent of `<VerticalScrollContainer>`, not a child of it.